elements/rh-card/demo/color-context.html

```diff
@@ -14,9 +14,9 @@ <h2 slot="header">Default card, slotted content and footer</h2>
   <rh-card id="custom">
     <h2 slot="header">Slotted title, content, and footer</h2>
     <label for="picker">Change this card's color-palette</label>
-    <rh-context-picker id="picker"
-                       target="custom"
-                       allow="darkest, lighter, lightest"></rh-context-picker>
+    <rh-context-picker id="picker" target="custom"></rh-context-picker>
+    <p>If a card inherits one of the six color palettes from a parent component, it will have either a lightest or darkest background.</p>
+    <p>If a card has a color palette set on itself, the card's background will correspond with the color palette's value.</p>
```

Comment:
```suggestion
    <rh-context-picker id="picker" target="custom"></rh-context-picker>
```
prose should be in docs, not demos

Reply:
Resolved in [4b1f395](https://github.com/RedHat-UX/red-hat-design-system/pull/2329/commits/4b1f3958250506f034bb5546725fb9902e0ed6ea).